tests/rptest/archival/abs_client.py

```diff
@@ -72,7 +72,9 @@ def delete_bucket(self, name: str):
         blob_service = BlobServiceClient.from_connection_string(self.conn_str)
         blob_service.delete_container(name)
 
-    def empty_bucket(self, name: str):
+    def empty_bucket(self, name: str, parallel=False):
+        # TODO: implement `parallel` as/when we start running any scale tests
+        # on Azure
```

Comment:
nit: this is going to be happening any day now. The Python ABS api supports parallel deletes via `delete_blobs` (max of 256 at a time) so it should be a small change. Can be done in a separate PR though.